java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -357,32 +363,34 @@ public void addExceptionalYield(SymbolicValue target, ProgramState exceptionalSt
     return stateStream.collect(Collectors.toList());
   }
 
-  private static boolean parametersAreNullable(MethodTree methodTree) {
-    return isAnnotatedWith(methodTree, "javax.annotation.ParametersAreNullableByDefault");
+  private static boolean isAnnotatedWith(MethodTree methodTree, String annotation) {
```

Comment:
This one probably deserve a better name, as it his checking in enclosing class and package.